### ISSUE TITLE
Preventing self search?

### ISSUE THREAD

**Issue by aderzelle** [bioinf/Sibelia#175]
Hello, 

Is there an option to prevent Sibelia to find synteny block inside one of the 2 input genomes? 
For example, when I compare genome_1 and genome_2, Sibelia will not only find the syntenic regions of the 2 genomes but will also find the duplications inside each genome. 

I think it would be interesting to have an option to avoid that, so that we could just get the synteny blocks involving the 2 genomes. The "sharedonly" option doesn't give me what I want, because if the genome is polyploid then it returns empty files.

**Comment by ilyaminkin** [comment]
What should the output be if genomes are polyploid? For example, suppose you have two genomes:

G1: +A +A +B +B
G2: +A +B +A +B

Where A and B ary synteny blocks. So what would be the desired output in this case?

**Comment by aderzelle** [comment]
It would be, if we just consider the first "A" of the G1 genome

G1 +A and G2 +A
G1 +A and G2 +B
etc ...
And never G1 + A and G1 +B 

If I understood correctly your example.

**Comment by ilyaminkin** [comment]
This is tricky because it is not clear why we should choose the first appearance of "A" as the "main" one, and not the second. There is no ready-made option to impose this behaviour in Sibelia. You can write a simple postprocessing script that takes ouput of Sibelia and "resolves" duplications the way you like.

**Comment by aderzelle** [comment]
Yes indeed, this is the approach I have undertaken. I am working on a very unusual organism for which no genomics tool is really well adapted (the bdelloid rotifers, which are degenerate polyploid with no homologuous chromosomes, which means I have to work with all allelic pairs or paralogs present. There is also no way to distinguish an allelic pair from a paralog).  Sibelia is nevertheless the best so far, I really like your program.

**Comment by ilyaminkin** [comment]
That is cool :) I am glad that you found Sibelia useful.
